cs/configuring.texy

```diff
@@ -279,99 +274,20 @@ nette:
 \--
 
 
-Low-level úpravy
----------------
-
-Všechna tato nastavení ovlivňují podobu výsledného systémového DI kontejneru. Oproti přímému zápisu do sekcí `services` či `factories` nabízejí srozumitelnější a stručnější syntax. Nicméně zůstává možnost si jednotlivé služby ještě "doladit" na nižší úrovni:
-
-/--neon
-services:
-	nette.mailer:
-		class: MySmtpMailer
-
-	nette.presenterFactory:
-		class: MyPresenterFactory
-\--
-
-Nebo vytvořit nové služby poděděním existujících:
-
-/--neon
-services:
-	myPresenterFactory < nette.presenterFactory
-		setup:
-			- setMapping([])
-\--
-
-
-
-
 Vlastní služby
 ==============
 
-Konfigurační soubor je místem, kam umísťujeme definice vlastních služeb. Slouží k tomu sekce `services`. Například tato definice služby:
-
-/--code neon
-services:
-	database: Nette\Database\Connection(%dsn%, %user%, %password%)
-
-# nebo ve dvou řádcích:
-	database:
-		factory: Nette\Database\Connection(%dsn%, %user%, %password%)
-
-# nebo ve třech řádcích :-)
-	database:
-		class: Nette\Database\Connection
-		arguments: [%dsn%, %user%, %password%]
-\--
+TODO: zmínit kontejner
 
-Vygeneruje:
 
-/--php
-	function createServiceDatabase()
-	{
-		return new Nette\Database\Connection(
-			$this->parameters['dsn'],
-			$this->parameters['user'],
-			$this->parameters['password']
-		);
-	}
-\--
-
-
-Definice služby:
+Konfigurační soubor je místem, kam umísťujeme definice vlastních služeb. Slouží k tomu sekce `services`. Například tato definice služby:
 
 /--code neon
 services:
-	database:
-		class: Nette\Database\Connection
-		factory: DbFactory::createConnection
-
-\--
-
-`DbFactory::createConnection`:
-
-/--php
-class DbFactory
-{
-	static function createConnection(Nette\DI\Container $container)
-	{
-		...
-	}
-}
-\--
-
-Vygeneruje:
-
-/--php
-	function createServiceDatabase()
-	{
-		return DbFactory::createConnection($this);
-	}
+	myService: App\Service\MyService(%argument1%, %argument2%)
```

Comment:
:+1: díky, to zní dobře. S tím překladem formulářů máš na mysli to pak např. porpojit s FormFactory?